bins/recipe-runner/src/runner.c

```diff
@@ -605,7 +607,12 @@ GglError runner(const RecipeRunnerArgs *args) {
         &resp
     );
     switch (ret) {
+    case GGL_ERR_NOMEM:
+        GGL_LOGW("Failed to get network proxy url from config - lack of memory."
+        );
+        break;
     case GGL_ERR_NOENTRY:
+        GGL_LOGW("Failed to get network proxy url - no such entry.");
```

Comment:
This is not a failure. Probably log debug and like "No network proxy set" or something.

Reply:
Done.